spec/github_helper_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper.rb'
+require 'webmock/rspec'
+
+describe Fastlane::Helper::GithubHelper do
+  describe 'download_file_from_release' do
+    it 'fails if it does not find the right release on GitHub' do
+      stub = stub_request(:get, 'https://raw.githubusercontent.com/repo-test/project-test/1.0/test-file.xml').to_return(:status => [404, "Not Found"])
```

Comment:
nit / brain dump: this makes me wonder: what if there's a 302 or something like that? Is that automatically handled by `open-uri`? (though the only case I can think of for a 302 in that case is if we were to rename the repository, in which case GitHub auto-redirects URLs pointing to the old repo name to point to the new one. A use case that we're unlikely to encounter anytime soon… but just curious)